packages/components/pages/f-checkout/src/components/Notes.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+    <div v-if="isSplitNotesEnabled">
+        <accordion
+            :id="noteTypeCourierOrOrder"
+            :title="$t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.title`)">
+            <span :class="$style['c-checkout-accordion-help']">{{ $t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.text`) }}</span>
+            <form-field
+                input-type="textarea"
+                :placeholder="$t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.placeholder`)"
+                :value="noteValue"
+                cols="30"
```

Comment:
could you have `cols`, `rows`, `maxlength` and any other prop thats reused here as a computed and then `v-bind="textAreaProps"`